.env.example

```diff
@@ -0,0 +1,3 @@
+WORKOS_ADMIN_PORTAL_HOSTNAME=setup.workos-test.com
```

Comment:
Adding environmental variable support so that we can setup a staging environment.